src/OrbitQt/CallTreeWidget.cpp

```diff
@@ -170,28 +170,87 @@ void CallTreeWidget::ResizeColumnsIfNecessary() {
   column_resizing_state_ = ColumnResizingState::kDone;
 }
 
-static std::string BuildStringFromIndices(const QModelIndexList& indices) {
-  std::string buffer;
+static std::string BuildStringFromIndices(QTreeView* tree_view, const QModelIndexList& indices) {
+  // The order of rows in the selection model is not the order in which the rows are displayed.
+  // We can recover that order by sorting using as comparison function for two rows the
+  // lexicographic order of the vector of the row indices from the root to those rows.
```

Comment:
nit: I understand the code, but I don't understand this sentence. Maybe make 2 or 3 sentences out of it, or alternatively some commas

Reply:
Eh, fair enough. I'm wondering if this comment is of any use at all, but in the meantime I rephrased it, I hope it's understandable now.

Reply:
Definitely better! Thanks